webapp/app/model/form_data.py

```diff
@@ -100,6 +106,7 @@ class MandatoryFormData(BaseModel):
     person_b_blind: Optional[str]
     person_b_gehbeh: Optional[str]
     person_b_has_disability: Optional[str]
+    person_b_requests_pauschbetrag: Optional[str]
```

Comment:
I am wondering if we need to require this field in the following scenario and set a validator for this:
`Person B is shown & Person B has a disability & Person B would have the right to request a pauschbetrag`

Reply:
I don't get it... what do you mean with: `Person B would have the right to request a pauschbetrag`

Reply:
There are situations where the Merkzeichenpage is shown but the person is not eligible for a Pauschbetrag (e.g. if everything is left empty).

Reply:
Ah so I should add it? In the end it's just a model validator like TestPauschBetragPersonAValidation. We can add another validation for the Merkzeichenpage if we need it?